pkg/tools/etcd_helper_test.go

```diff
@@ -689,14 +689,12 @@ func TestGetEtcdVersion_ValidVersion(t *testing.T) {
 	}))
 	defer testServer.Close()
 
-	var relVersion string
-	var intVersion string
+	var version string
 	var err error
-	if relVersion, intVersion, err = GetEtcdVersion(testServer.URL); err != nil {
+	if version, err = GetEtcdVersion(testServer.URL); err != nil {
 		t.Errorf("Unexpected error: %v", err)
 	}
-	assert.Equal(t, "2.0.3", relVersion, "Unexpected external version")
-	assert.Equal(t, "2", intVersion, "Unexpected internal version")
+	assert.Equal(t, "{\"releaseVersion\":\"2.0.3\",\"internalVersion\":\"2\"}", version, "Unexpected version")
```

Comment:
This doesn't seem to make sense any more.  You no longer have release and internal versions?